src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPane.java

```diff
@@ -127,10 +130,10 @@ protected SecondaryToolbar createSecondaryToolbar()
       replaceToolbar_ = new SecondaryToolbar("Replace");
       replaceMode_ = true;
 
-      replaceLabel_ = new Label();
+      replaceLabel_ = new Label("Replace with: ");
```

Comment:
This needs to be a `FormLabel`. The `Label` class, despite the name, creates a `div` instead of an HTML label, and there is a significant difference for accessibility. You can tell just by clicking on the label in the UI. If focus goes to the associated control (the textbox), then it is properly associated such that a screen reader would understand but that doesn't happen here.

To fix, make this a `FormLabel`, and create it AFTER the `replaceTextBox_` (but before adding either of them to the `replaceToolbar_`). Use the FormLabel constructor that takes both the string and the control to associate with, i.e.:
```
      replaceTextBox_ = new TextBox();
      replaceLabel_ = new FormLabel("Replace with: ", replaceTextBox_);
      replaceToolbar_.addLeftWidget(replaceTextBox_);
      replaceToolbar_.addLeftWidget(replaceLabel_);
```
This will generate a unique element ID for the text box, and then set `for=ID` attribute on the `<label>` thus establishing the connection.

Reply:
Fixed!